services/greasyfork/greasyfork-license.service.js

```diff
@@ -0,0 +1,34 @@
+import { renderLicenseBadge } from '../licenses.js'
+import { InvalidResponse } from '../index.js'
+import BaseGreasyForkService from './greasyfork-base.js'
+
+export default class GreasyForkLicense extends BaseGreasyForkService {
+  static category = 'license'
+  static route = { base: 'greasyfork', pattern: 'l/:scriptId' }
+
+  static examples = [
+    {
+      title: 'Greasy Fork',
+      namedParams: { scriptId: '407466' },
+      staticPreview: renderLicenseBadge({ licenses: ['MIT'] }),
+    },
+  ]
+
+  static defaultBadgeData = { label: 'license' }
+
+  transform({ data }) {
+    if (data.license === null) {
+      throw new InvalidResponse({
+        prettyMessage: 'version not found',
```

Comment:
Should this be license instead?

```suggestion
        prettyMessage: 'license not found',
```